libs/remix-solidity/src/compiler/compiler.ts

```diff
@@ -275,6 +277,9 @@ export class Compiler {
 
     this.state.worker.addEventListener('message', (msg: Record <'data', MessageFromWorker>) => {
       const data: MessageFromWorker = msg.data
+      if (this.state.compilerRetriggerMode == CompilerRetriggerMode.retrigger && data.timestamp !== this.state.compilationStartTime) {
```

Comment:
see the function above `loadInternal` this is called when the compiler is loaded is a old fashionned way (using script tag), probably that this function should be updated too.

Reply:
I assume that function doesn't do any import handling? in that case it would have no use.

Reply:
you mean solidity import?
once a compiler is loaded with loadInternal (which should not actually happen a lot anymore), solidity import should still function..